pkg/ttnpb/field_mask_validation.go

```diff
@@ -36,6 +36,7 @@ var isEndDeviceReadFieldPaths = []string{
 	"version_ids.firmware_version",
 	"version_ids.hardware_version",
 	"version_ids.model_id",
+	"picture",
```

Comment:
If this is here, then all subfields have to be present as well.

Reply:
I think this is actually fine the way it is. I can't think of any reason why a user would only want to get the MIME type of the picture.